tools/istio-iptables/pkg/cmd/run_test.go

```diff
@@ -470,3 +473,42 @@ func TestHandleInboundPortsIncludeWithWildcardInboundPortsAndTproxy(t *testing.T
 		t.Errorf("Output mismatch\nExpected: %#v\nActual: %#v", expectedIpv4Rules, ip4Rules)
 	}
 }
+
+func TestRun(t *testing.T) {
```

Comment:
minor nit: Might want to give this test a different name. A bit descriptive name or comment on what this test is trying to cover. 
Easier later for someone else to debug a failure.